packages/firestore/package.json

```diff
@@ -49,16 +49,22 @@
   },
   "exports": {
     ".": {
-      "node": "./dist/index.node.cjs.js",
+      "node": {
+        "require": "./dist/index.node.cjs.js",
+        "import": "./dist/index.node.mjs"
+      },
       "default": "./dist/index.esm2017.js"
     },
     "./lite": {
-      "node": "./dist/lite/index.node.cjs.js",
+      "node": {
+        "require": "./dist/lite/index.node.cjs.js",
+        "import": "./dist/lite/index.node.esm2017.js"
```

Comment:
import needs to point to a file with `mjs` extension.

Reply:
Oops, fixed.